lib/driver.js

```diff
@@ -468,6 +470,12 @@ class IosDriver extends BaseDriver {
     }
   }
 
+  async startIWDP () {
+    if (this.opts.startIWDP) {
+      await IWDP.start(this.opts.udid);
```

Comment:
Does this work? Treating this as a static method? We don't want to be able to keep track of the object? Close iwdp when done?